README.md

```diff
@@ -17,138 +19,143 @@ Before getting started with this project, please get yourself familiar with the
 - [The Yocto Project Software Overview](https://www.yoctoproject.org/software-overview/)
 - [The Device Update for IoTHub Overview](http://github.com/azure/iot-hub-device-update)
 
-## Getting Started
-
-- [Clone The Source Code](#clone-source-code)
-- [How Building The Project Locally](#how-to-build-the-project-locally)
-- [How To Build The Project using Azure DevOp Build Pipeline](#how-to-build-the-project-using-azure-devop-build-pipeline)
-
-
 ### Get Source Code
 
-Please note that, at the time of this writing, we only support `honister` release of the Yocto Project. Keep in mind the following environment variables that will be referenced throughout this document:
+Please note that, at the time of this writing, we only support `kirkstone` release of the Yocto Project. 
 
+The following variables are referenced in the below section setting up the build. 
 
 | Variable Name | Description |
 |---|---|
-| $yocto_release | A name of the version of the Yocto Project used to build the images.<br/>(Only support `honister` at the moment) |
-| $proj_root | A root directory where this project will be cloned into.|
+| $yocto_release | A name of the version of the Yocto Project used to build the images.<br/>(Only support `kirkstone` at the moment) |
+| $project_root  | A root directory where this project will be cloned into.|
+| $adu_release   | The release of Device Update you're planning on using (should default to `'main'`) | 
 
+You can either just include the string wholesale in the terminal (eg for `$yocto_release` just use `'kirkstone'`) or set the variable at the beginning and then copy the command from this screen.
 
+You can set a bash variable like `yocto_release` this:
+```shell
+yocto_release=kirkstone
+```
+and for `adu_release` like this: 
+```shell
+adu_release=main
+```
+and for `project_root` like this:
+```shell
+project_root=~/
+```
 
-- Clone the Yocto (Poky) project
-    
-    ```shell
-    yocto_release=honister
-    adu_release=main
-    
-    # Clone project with Yocto configuration files
-    git clone <github url> -b <branchname> $proj_root
-
-    cd $project_root/yocto
+The first step for building the project is cloning this repository onto your device using the following command:
 
-    # Clone the Yocto Project (poky) into 'yocto' dir
-    git clone --depth 1 --branch $yocto_release git://git.yoctoproject.org/poky
-    ```
+1. Clone this repository onto your device:
+    
+```shell
+git clone https://github.com/Azure/iot-hub-device-update-yocto -b <branchname> $proj_root
+```
+2. Once you've cloned the project you next need to change into our "working directory" where the individual layers (in Yocto these layers build up to an image like a cake... or foundation.. or cake). 
```

Comment:
For this: _(in Yocto these layers build up to an image like a cake... or foundation.. or cake)._ I'm not sure what the second "cake" is for? Otherwise I think the example works.

Reply:
I dunno. I thought it was funny. The joke comes over better when it's announced instead of read. Or if someone knows how I talk.  I think this used to have a comparison to an ez bake oven thing before. Some of this was written while I was a bit too tired. 

I'll edit it to make more sense.